coalahtml/_coalahtml/app/controllers/files.js

```diff
@@ -3,36 +3,49 @@
 angular.module('coalaHtmlApp')
   .controller('FilesCtrl', function ($scope, $http, $routeParams) {
     var parseCoalaProject = function(fileName, projectDir, filesJSON) {
-      /* Expected file structure in filesJSON:
-       * JSON object with keys as directory relative to the project
-       * path and values are files inside directory relative to the
-       * project.
-       * For files in the project root, the key should be "".
-       */
-      filesJSON = filesJSON || projectDir + '/files.json';
-      $scope.data = [];
-      $scope.fileBack = fileName.split("/").slice(0, -1).join('/');
-
-      $http.get(filesJSON).success(function(fileData){
+        /*
+        Expected file structure in filesJSON:
+        JSON object with keys as directory relative to the project
+        path and values are files inside directory relative to the
+        project.
+        For files in the project root, the key should be "".
+        */
+        filesJSON = filesJSON || projectDir + '/files.json';
+        $scope.data = [];
+        $scope.fileBack = fileName.split("/").slice(0, -1).join('/');
+        if($scope.fileBack + "/" === $scope.roothome) {
+            $scope.fileBack += "/";
+        }
+        $http.get(filesJSON).success(function(fileData) {
```

Comment:
I (once again) don't understand the indentations used ...
PLEASE decide on a codestyle !
If this block is moved 2 spaces inward, the `$http` and `if` are on the same column. That's confusing isnt it ?